pages.pt_BR/linux/adig.md

```diff
@@ -0,0 +1,24 @@
+# adig
+
+> Imprime informações recebidas dos servidores do Sistema de Domínio de Nome(DNS).
+> Mais informações: <https://manned.org/adig>.
+
+- Exibe uma gravação A (padrão) do DNS por nome(s) de servidor(es):
+
+`adig {{examplo.com}}`
+
+- Exibe uma saída de [d]epuração extra:
+
+`adig -d {{examplo.com}}`
+
+- Conecta ao servidor DNS [e]specificado:
+
+`adig -s {{1.2.3.4}} {{examplo.com}}`
+
+- Usa porta TCP específica para conectar ao servidor DNS:
+
+`adig -T {{port}} {{examplo.com}}`
+
+- Usa porta UDP específica para conectar ao servidor DNS:
+
+`adig -U {{port}} {{examplo.com}}`
```

Comment:
```suggestion
- Use uma porta UDP específica para se conectar ao servidor DNS:

`adig -U {{port}} {{examplo.com}}`
```

The same issue as above.
Note: `Use a specific UDP port to connect to DNS server:` can be used a translation source.